common-lib/src/main/resources/application.conf

```diff
@@ -20,4 +20,4 @@ es {
 }
 
 # PROD only stuff (play secret etc)
-include "/etc/gu/grid-prod.properties"
\ No newline at end of file
+include file("/etc/gu/grid-prod.properties")
```

Comment:
We weren't reading this file without `file(`. Not quite sure how TEST and PROD is happy with this...

Reply:
Is this safe to add in that case?

Reply:
It should be. Will create a branch with just this change and run it in TEST to check.

Reply:
https://github.com/guardian/grid/pull/2933